src/SelectBox/SelectBox.styles.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
```

Comment:
I like the idea of having styled-components separated, but if you take a look at the codebase this isn't a pattern we're using here.

What do you think about moving those styled-components into the same file we have for this component?

Reply:
Thank you so much for pointing it out, i don't have a lot of experience with styled components and at first it sounds strange to write other components in my component file, but i just saw some examples on the project and i think that's also a nice design. I will apply it asap.

Reply:
@luizbaldi actually we have one component that uses this approach and that's `Select` component. it was because there was so much styling going on it was way too much to keep it all in one file. if there's lots of styles for SelectBox too we can keep it this way